sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1082,6 +1084,8 @@ def task_worker(self, stopping_event, sfp_error_event, y_cable_presence):
 
                             if value == sfp_status_helper.SFP_STATUS_INSERTED:
                                 helper_logger.log_info("Got SFP inserted event")
+                                # Delay for I2C MgmtInit to complete
+                                time.sleep(MGMT_INIT_TIME_DELAY_SECS)
```

Comment:
as we found in the other thread, let's not sleep here. let's put them into a list, and later look the list to get the data. let's fix this properly.

Reply:
I didn't understand

Reply:
Removed the cumulative delay across port insertions